### DISCUSSION TITLE
Communities Join Screens - Implement request denied of Context Drawer options

### DISCUSSION THREAD

**Discussion by J-Son89** [status-im/status-legacy#16665] [General]
Following from https://github.com/status-im/status-mobile/issues/14464

The option for request denied still needs to be implemented. This can be found in the file - 
src/status_im2/contexts/communities/menus/community_options/view.cljs

The designs are already implemented however the trigger from the backend needs to be setup on the sub for the community.

**Comment by J-Son89** [comment]
Hi @alwx there is discussion whether this is needed/ possible on mobile. Let's sync up with @pedro-et and @cammellos about this before starting!

**Comment by alwx** [comment]
Could you update me on a status of it? cc @pedro-et @cammellos

**Comment by J-Son89** [comment]
Pending the output of this discussion we should also close this issue https://github.com/status-im/status-mobile/issues/14460 if we do not want the request denied flow in mobile.
